mainDarwin.goreleaser.yml

```diff
@@ -1,14 +1,19 @@
+version: 2
+
 builds:
   - main: ./cmd/tester/main.go
     binary: tester
     env:
-      - CGO_ENABLED=0
+      - CGO_ENABLED=1
```

Comment:
@ryan-gang with CGO enabled I think we're going to run into errors running this on alpine. That's why we've have CGO set to 0 previously. Can you confirm that this works fine on alpine?